### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by sunga12**
- Fixed horizontal scroll 
- Added the mobile menu design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,123 +7,131 @@
...
+          <li><a href="#About">About</a></li>
+          <li><a href="#Contact">Contact</a></li>
+          </ul>
+          </div>
         <div class="name menuicon">
-        <img class="hamburger" src="./images/Hamburger.png" alt="menu icon">
+          <img class="hamburger" src="./images/Hamburger.png" alt="menu icon">
         </div>
-    </nav>
-</header>
```
Comment (emmanuellekamwa) [inline index.html:21]: - [ ] On the mobile view I can see a horizontal scroll bar at the bottom of the page meaning that your page is overflowing. Please fix that.
- [ ] When I click on the hamburger button, noting pops up. Please fix that.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ⛔❌🛑**
Hello @sunga12 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks.

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (sunga12) [reply index.html:21]: Kindly elaborate the points brought up. Thanks in advance.

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @sunga12, and @Kaghenimbale👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**